LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2014, 2015 ClusterHQ
+   Copyright 2014-2015 ClusterHQ
```

Comment:
Seeing as this is a copyright notice, I suspect this should be "ClusterHQ Inc", seeing as that's the actual entity name that owns Flocker.